src/inspect_ai/_cli/eval.py

```diff
@@ -270,13 +270,13 @@ def eval_options(func: Callable[..., Any]) -> Callable[..., click.Context]:
         "--logprobs",
         type=bool,
         is_flag=True,
-        help="Return log probabilities of the output tokens. OpenAI, TogetherAI, and Huggingface only.",
+        help="Return log probabilities of the output tokens. OpenAI, TogetherAI, Huggingface and llama-cpp-python only.",
```

Comment:
I'm okay merging this now, in that case let's just remove the docs on anything not supported by the chat api for the time being

Reply:
Sounds good. Will remove this, and add it back later once upstream support is fixed.